tasks/copy_files_to_archive/copy_files_to_archive.py

```diff
@@ -79,7 +79,8 @@ def task(
                     a_file[INPUT_SOURCE_BUCKET_KEY],
                     a_file[INPUT_SOURCE_KEY_KEY],
                     a_file[INPUT_TARGET_BUCKET_KEY],
-                    a_file.get(INPUT_MULTIPART_CHUNKSIZE_MB, None) or default_multipart_chunksize_mb,
+                    a_file.get(INPUT_MULTIPART_CHUNKSIZE_MB, None)
+                    or default_multipart_chunksize_mb,
```

Comment:
If this needs to be on a second line, can we indent it?

Reply:
Auto formatting caught  it, I can change it back to a single line